Abstract Syntax and Semantics Document/index.html

```diff
@@ -497,6 +490,107 @@
+      Those concepts are not defined in this draft, and instead we include some RDF stream subclasses defined by certain constraining properties.
+      This section may be expanded with more subclasses if a need is identified.</p>
+    
+    <h3>Time-bounded RDF Streams</h3>
+    
+      <p>A <em>time-bounded-above RDF stream</em> is an RDF Stream where for every timestamp predicate that occurs in the stream, there is a temporal entity in its range that bounds from above (is greater than or equal to) all timestamps of that predicate in the stream.
+      </p>
+      
+      <p>A <em>time-bounded-below RDF stream</em> is an RDF Stream where for every timestamp predicate that occurs in the stream, there is a temporal entity in its range that bounds from below (is less than or equal to) all timestamps of that predicate in the stream.
+      </p>
+      
+      <p>A <em>time-bounded RDF stream</em> is an RDF Stream that is time-bounded above and time-bounded below.
+      </p>
+      
+        <p  class="ednote">
+          Consider the class of timestamp predicates whose associated poset of temporal entities is <em>pairwise bounded</em>, 
+          i.e. has the property that every pair of temporal entities is bounded, 
+          i.e. there exists a temporal entity in the poset that is greater or equal to both of them and another temporal entity that is less or equal to both of them.
+          
+          This is a fairly natural property to expect for partial orders on timescales (the timescale is the set of temporal entities, 
+          alternate terminology "time-plenum"
+          as in <a href="http://www.ihmc.us/users/phayes/docs/timeCatalog.pdf">http://www.ihmc.us/users/phayes/docs/timeCatalog.pdf</a>)
+          with "linear topology" (e.g. [[OWL-TIME]]).
+          However it does not hold in the case of branching timescales 
+          (see <a href="http://www.ihmc.us/users/phayes/docs/timeCatalog.pdf">http://www.ihmc.us/users/phayes/docs/timeCatalog.pdf</a>),
+          because there are incomparable branches (called paths), which may be considered, e.g. as different worlds (in the sense of Kripke frames), scenarios (planning usecase),
+          or concurrent processes (process modelling).
+          
+          It would be a useful property if the class of time-bounded RDF streams were closed under stream merger and union,
+          and this property would hold, under the current definition, in the case of pairwise-bounded timescales,
+          because we could find bounds for the temporal entities of each timestamp predicate by taking an upper bound of the pair of upper bounds from the two streams, 
+          and similarly for the lower bounds.
+          
+          However, the closure property does not hold in the case of a branching temporal topology under the above definition of time-bounded. 
```

Comment:
so, this consideration would lead to a slightly modified version of the time bounded substreams?